zerver/tests/test_hooks.py

```diff
@@ -1507,3 +1507,16 @@ def test_semaphore_deploy(self):
     def get_body(self, fixture_name):
         # type: (text_type) -> text_type
         return self.fixture_data("semaphore", fixture_name, file_type="json")
+
+class HelloWorldHookTests(WebhookTestCase):
+    STREAM_NAME = 'test'
+    URL_TEMPLATE = "/api/v1/external/helloworld?&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'hello'
+
+    def test_hello_message(self):
+        expected_subject = u"Hello World";
+        expected_message = u"Hello! I am happy to be here! :smile: \nThe Wikipedia featured article for today is **[Marilyn Monroe](https://en.wikipedia.org/wiki/Marilyn_Monroe)**";
+        self.send_and_test_stream_message('hello', expected_subject, expected_message,content_type="application/x-www-form-urlencoded")
+
+    def get_body(self, fixture_name):
+        return self.fixture_data("helloworld", fixture_name, file_type="json")
```

Comment:
I think it would be better to have the actual code for the helloworld webhook be its own commit in this PR, so that you can link to the commit in the Zulip history directly, and it'll only have the webhook itself (and not these new docs) shown.